src/runtime/util.ts

```diff
@@ -66,3 +66,11 @@ export function setDiffCustom<T, U>(from: T[], to: T[], keyFn: (T) => U): {add:
   }
   return result;
 }
+
+/**
+ * A hack to ignore a floating promise and bypass the linter. Promises should *not* be ignored. They should always be awaited or returned. See
```

Comment:
I would rather say promises should rarely be left floating and when such behaviour is intended, they should be clearly marked as such.

Reply:
done